services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/process_message_running_service_setup.py

```diff
@@ -32,7 +32,7 @@ async def _subscribe_to_rabbitmq(app) -> str:
 def on_app_startup(app: FastAPI) -> Callable[[], Awaitable[None]]:
     async def _startup() -> None:
         with log_context(
-            _logger, logging.INFO, msg="setup resource tracker"
+            _logger, logging.INFO, msg="RUT process message running service startup.."
```

Comment:
i dont get that string, what is "process message running service"

Reply:
the logline before was lear to me ;)

Reply:
improved